woocommerce/woocommerce
# [Experimental] Product Filters Redesign > Status filter block: Rename Stock Status Filter to Status Filter

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51859 

This PR introduces the following updates related to renaming the “Stock” filter to “Status” filter:

1. Block Title and Description updated: The block’s title and description now reflect the new “Status” name.
2. Slug Renaming: The block’s slug has been updated from “product-filter-stock-status” to “product-filter-status”.
3. Folder Name: The folder containing the block files has been renamed to align with the updated block name.
4. Icon Name and Webpack Entry: Both the block’s icon and Webpack entry point have been renamed to reflect the new name.
5. Source Code References: All references to the “Stock Status” filter in the source code have been updated to "Status" filter.
6. Test Updates: An E2E test that was failing after the updates has been fixed.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

> [!NOTE]  
> **Pre-requisite**
> - If you're using WC Beta Tester plugin, make sure the `experimental-blocks` feature is enabled (Tools > WCA Test Helper > Features).

1. Navigate to Product Filters template (Appearance > Editor > Patterns > General > Product Filters (Experimental).
2. Click on it to open the template part editor.
3. Change to the Code Editor mode;
4. Replace the current code with the following:

```

<div class="wp-block-woocommerce-product-filters wc-block-product-filters">
<h3 class="wp-block-heading">Status Filter</h3>


</div>

```

5. Make sure that you can see the Status filter being rendered in the canvas.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment  Rename the Stock Status filter block to Status block

</details>
